content/release.rst

```diff
@@ -19,6 +19,19 @@ Releasing a product usually consists of publishing the following:
 * Documentation, which includes Release Notes, Known Issues and Upgrade Steps.
 * Public announcement. Notification email or website news/blog article.
 
+Each version sent to a customer should have an unique version number and
+should be part of the release notes history.
+
+When a functionality is released for a certain customer group, but is not
+wanted in the main release series, it should be released from a branch which
+is not removed, a tag is created for each release and all release notes
+are imported into the main branch.
```

Comment:
So many things to be improved in the above paragraph that I have rephrased it entirely. Not sure about the last phrase, please check that I got it right.

_Sometimes, special features are required by clients with custom needs and the functionalities are not to be included in the main release series. The versions with dedicated features should be released from their own branches, which are not to be removed. Each release from these branches should have an associated **tag**. All the related custom bits in the release notes are to be imported into the release notes from the main branch though._